src/main/java/net/tammon/sip/packets/Head.java

```diff
@@ -60,13 +60,16 @@ public int getMessageType() {
         return messageType;
     }
 
-
-
     public void setMessageType(int messageType) {
         this.messageType = messageType;
     }
 
     public byte[] getDataAsByteArray (){
         return Data.getByteArray(this.transactionId, this.messageType);
     }
+    
+    public static int getFixLength() {
+        //     sizeof(transaction id) + sizefo(messagetype)
```

Comment:
Remove unneeded lines
```suggestion
        // The length is the size of the attribute and the data length which are both Integers
```